src/js/components/Video.js

```diff
@@ -152,19 +205,28 @@ export default class Video extends Component {
       );
     }
 
+    var onClickControl = this._onClickControl;
+    if (this.props.onClick) {
+      onClickControl = this.props.onClick;
+    }
+
     return (
       <div className={classes.join(' ')} onMouseMove={this._onMouseMove}>
         <video ref="video" poster={this.props.poster}>
           {this.props.children}
         </video>
-        <div className={`${CLASS_ROOT}__summary`}>
-          <Button className={`${CLASS_ROOT}__control`} plain={true}
-            primary={true}
-            onClick={this._onClickControl}>
-            {controlIcon}
-          </Button>
-          {title}
-        </div>
+        <Box pad="none" align="center" justify="between" className={`${CLASS_ROOT}__summary`}>
+          {videoHeader}
+          <Box pad="large" align="center" justify="center">
+            <Button className={`${CLASS_ROOT}__control`} plain={true}
```

Comment:
Can we use the same Button + Icon structure as we have on top of this file? Please make sure to include the proper a11yTitle with some good description of what is this button is doing